README.md

```diff
@@ -0,0 +1,23 @@
+
+## Phoenicis PlayOnLinux
+
+Phoenicis PlayOnLinux is a graphical front-end for Wine.
+Install and run Windows software on Linux
+
+
+## Installation Instruction :
```

Comment:
Typo: Space before the colon. Also, I think that it should be "Instructions", not "Instruction".